hil/ext/obm/ipmi.py

```diff
@@ -68,10 +68,14 @@ def _ipmitool(self, args):
         return status
 
     @no_dry_run
-    def power_cycle(self):
+    def power_cycle(self, force=False):
```

Comment:
I would *not* have a default for force here (in the API call and client library it makes sense). We're only invoking this from the one place, so having the default just introducess pointless complexity. Also, the mock driver doesn't have the default, so the APIs aren't actually the same, which is a problem.